templates/settings.php

```diff
@@ -5,6 +5,7 @@
 global $wp_roles;
 
 $product_attributes = function_exists('wc_get_attribute_taxonomies') ? wc_get_attribute_taxonomies() : '';
+$shop_analytics_product_id = get_option('shop_analytics_product_id', 'post_id');
```

Comment:
I think it's fine to remove the `shop_analytics` prefix for the variable name as we are already in the `ShopAnalytics` namespace. The option name should remain the same however

```suggestion
$product_id = get_option('shop_analytics_product_id', 'post_id');
```